lib/std/enums.zig

```diff
@@ -1086,6 +1086,14 @@ pub fn IndexedMap(comptime I: type, comptime V: type, comptime Ext: ?fn (type) t
             return &self.values[index];
         }
 
+        /// Gets the address of the const value associated with a key.
+        /// The key must be present in the map.
+        pub fn getPtrConstAssertContains(self: *const Self, key: Key) *Value {
```

Comment:
```suggestion
        pub fn getPtrConstAssertContains(self: *const Self, key: Key) *const Value {
```
shouldn't this return a const pointer?

Reply:
:bow: Updated

(Sorry about that, untested)